src/services/dashboards/widgets/cost-map/CostMapWidget.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <widget-frame v-bind="widgetFrameProps"
                   :error-mode="false"
+                  :widget-link="state.widgetLink"
```

Comment:
quibble: There's no `widgetLink` yet...!🫢 No errors?

Reply:
I need to comment  Thnx